_posts/2019-08-21-rust-bloat.markdown

```diff
@@ -86,7 +86,7 @@ For optimizing compile times in [unicode-normalization][unicode-normalization#37
 
 Compile time and executable size are aspects of performance (even though often not as visible as runtime speed), and [performance culture] applies. Always measure, using tools like [cargo-bloat] where appropriate, and keep track of regressions.
 
-A good case study for cargo-bloat is [clap](https://clap.rs/2018/01/09/new-years-weight-loss/), though it's still pretty heavyweight today (it accounts for about 1M of Zola's debug build, measured on macOS).
+A good case study for cargo-bloat is [clap](https://clap.rs/2018/01/09/new-years-weight-loss/) ([archived link](http://ghostarchive.org/archive/APVxZ)), though it's still pretty heavyweight today (it accounts for about 1M of Zola's debug build, measured on macOS).
```

Comment:
```suggestion
A good case study for cargo-bloat is [clap](https://web.archive.org/web/20191224123911/https://clap.rs/2018/01/09/new-years-weight-loss/), though it's still pretty heavyweight today (it accounts for about 1M of Zola's debug build, measured on macOS).
```

Reply:
Thanks for the heads-up. I think I'll go with archive.org, as I have confidence in its long term stability, especially as it's not an http link.

Reply:
the site in question has https but it doesn't force it, fyi. I like it because theres no unsolvable captcha on mobile.
Doesn't matter what you prefer, as long as the end goal is accomplished.
Thanks for merging.